moto/workspacesweb/responses.py

```diff
@@ -38,6 +38,26 @@ def browser_settings(request: Any, full_url: str, headers: Any) -> TYPE_RESPONSE
         else:
             return handler.delete_browser_settings()
 
+    @staticmethod
+    def user_settings(request: Any, full_url: str, headers: Any) -> TYPE_RESPONSE:  # type: ignore[misc]
+        handler = WorkSpacesWebResponse()
+        handler.setup_class(request, full_url, headers)
+        if request.method == "GET":
+            return handler.get_user_settings()
+        else:
+            return handler.delete_user_settings()
+
+    @staticmethod
+    def user_access_logging_settings(
+        request: Any, full_url: str, headers: Any
+    ) -> TYPE_RESPONSE:  # type: ignore[misc]
```

Comment:
```suggestion
    def user_access_logging_settings(  # type: ignore[misc]
        request: Any, full_url: str, headers: Any
    ) -> TYPE_RESPONSE:
```
Our formatter `ruff` isn't smart enough to understand where the `type: ignore` needs to go, so I've moved it manually to the right line. I haven't tested this locally, but I think that should fix it :crossed_fingers: